### PR TITLE
Use ignore_lang for flag and alternate links

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Specify more to ignore languages instead of just 1.

Can be used later on for the ignore build methode.

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -462,6 +464,10 @@ def alternate_link_tags(page)
     link_tags.join("\n    ")
   end
 
+  def lang_ignore?(page, locale)
+    page&.data&.lang_ignore&.include?(locale.to_s)
```
Comment (fatboypunk) [inline config.rb]: don't have to do page i guess, and are there pages without data?

Comment (RonaldDijkstra) [reply config.rb]: Okay. 

And there are pages without page.data.lang_ignore yes

Comment (fatboypunk) [reply config.rb]: yeah but also without page.data?

Comment (sn3p) [reply config.rb]: `page.data` always exists so could remove the first `&`.

Comment (RonaldDijkstra) [reply config.rb]: Ah cool thanks guys

Comment (RonaldDijkstra) [reply config.rb]: We need 'page'

Comment (fatboypunk) [reply config.rb]: it's not about the page it's about wether the safe operator is needed, you can use this:
`page.data.lang_ignore&.include?(locale.to_s)`

Comment (RonaldDijkstra) [reply config.rb]: Aight!

**Pull request merged**